### ISSUE TITLE
Issue single modal showing number for Manual issues and Best practice

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1560]
In the issue single modal best practice and manual testing issues are incorrectly showing the number that has been added for internal use only.

<img width="949" height="542" alt="Image" src="https://github.com/user-attachments/assets/87f99e82-0e93-4a1c-a8b7-48c93dde98b7" />
<img width="1003" height="609" alt="Image" src="https://github.com/user-attachments/assets/39cdb4d8-def0-4075-a041-6ca039a13591" />

These (correctly) do not show in Pro Open issues:

<img width="889" height="586" alt="Image" src="https://github.com/user-attachments/assets/b23e7351-1db8-422c-9968-2e25164a02e3" />

This number was added for sorting purposes and shouldn't be shown on the front end, as it's not a WCAG number and may cause confusion.

**Comment by amberhinds** [comment]
There was a 0.2, though I can't remember what it was at this point - maybe Non-WCAG Accessibility Issue. That one also should not show the number. Basically any number starting with a zero should be hidden from users.

**Comment by SteveJonesDev** [comment]
Fixed in the PR: [https://github.com/equalizedigital/accessibility-checker/pull/1563](<https://github.com/equalizedigital/accessibility-checker/pull/1563>)<br>Released in version 1.39.0
